src/contrib/loggraph/src/java/org/apache/zookeeper/graph/JsonGenerator.java

```diff
@@ -121,8 +121,8 @@ public JsonGenerator(LogIterator iter) {
 		} else if ((m = newElectionP.matcher(e.getEntry())).find()) {
 		    Iterator<Integer> iterator = servers.iterator();
 		    long zxid = Long.valueOf(m.group(2));
-		    int count = (int)zxid;// & 0xFFFFFFFFL;
-		    int epoch = (int)Long.rotateRight(zxid, 32);// >> 32;
+		    long count = zxid & 0xffffffffffL;
```

Comment:
How can this be all over the code base instead of a function somewhere in a util file